frontend/packages/volto-intranet/package.json

```
@@ -55,6 +56,7 @@
     "@kitconcept/volto-solr": "2.0.0-alpha.1",
     "@plone/components": "workspace:*",
     "classnames": "2.5.1",
+    "@kitconcept/volto-iframe-block": "2.2.0",
```

```suggestion
    "@kitconcept/volto-iframe-block": "workspace:*",
```
(then it uses the one installed by mrs.developer.json)